examples/compute_instance/disk_snapshot/main.tf

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+
+  version = "~> 3.0"
+}
+
+locals {
+  instance_disks = [for i in range(length(var.additional_disks)) : "projects/${var.project_id}/disks/instance-simple-001-${i + 1}/zones/${data.google_compute_zones.available.names[0]}"]
+}
+
+data "google_compute_zones" "available" {
+  project = var.project_id
+  region  = var.region
+}
+
+module "instance_template" {
+  source           = "../../../modules/instance_template"
+  region           = var.region
+  project_id       = var.project_id
+  subnetwork       = var.subnetwork
+  service_account  = var.service_account
```

Comment:
Just create the service account in the example instead of making it a variable.

Reply:
Done